corehq/apps/app_manager/templates/app_manager/partials/modules/case_search_property.html

```diff
@@ -140,26 +140,40 @@ <h4 class="modal-title">
                   </div>
                 </div>
                 </fieldset>
-                {% if js_options.has_geocoder_privs %}
-                    <fieldset data-bind="visible: appearanceFinal() == 'fixture' || appearance() == ''">
+                {% if js_options.has_geocoder_privs or js_options.exclude_from_search_enabled %}
+                    <fieldset>
                       <legend>{% trans "Advanced Options" %}</legend>
-                      <div class="form-group">
-                        <label class="control-label col-xs-12 col-sm-3">
-                          {% trans "Geocoder Receiver Expression" %}
-                        </label>
-                        <div class="col-xs-12 col-sm-9">
-                          <input class="form-control" type="text" data-bind="value: receiverExpression"/>
+                      {% if js_options.has_geocoder_privs %}
+                        <div data-bind="visible: appearanceFinal() == 'fixture' || appearance() == ''">
```

Comment:
@orangejenny This is the second option I came up with. The first was to add the same `visible` check to the two `form-group` divs separately, which felt repetitive.

Reply:
I think this is good. The only alternative that comes to mind is putting a `visible` binding on each of the two `form-group` divs, so then you're not adding an extra div but you're adding extra code, so it doesn't seem either better or worse.